common/static/coffee/src/discussion/views/discussion_thread_list_view.coffee

```diff
@@ -243,13 +243,14 @@ if Backbone?
       @trigger("thread:removed", thread_id)
 
     setActiveThread: (thread_id) ->
+      @$(".forum-nav-thread-link").find(".sr").remove()
       @$(".forum-nav-thread[data-id!='#{thread_id}'] .forum-nav-thread-link").removeClass("is-active")
-      @$(".forum-nav-thread[data-id='#{thread_id}'] .forum-nav-thread-link").addClass("is-active")
+      @$(".forum-nav-thread[data-id='#{thread_id}'] .forum-nav-thread-link").addClass("is-active").find(".forum-nav-thread-wrapper-1").prepend('<span class="sr">' + gettext("Current conversation") + '</span>')
 
     goHome: ->
       @template = _.template($("#discussion-home").html())
       $(".forum-content").html(@template)
-      $(".forum-nav-thread-list a").removeClass("is-active")
+      $(".forum-nav-thread-list a").removeClass("is-active").find(".sr").remove()
```

Comment:
I'd prefer if we had a specific class attribute for the current label (``.is-current`) rather than relying on the class of ``.sr`` in this place in the DOM. Can you add that to the HTML you're adding in https://github.com/edx/edx-platform/pull/6766/files#diff-2ad9e53cebc84971c8411b5a54b1198bR248 and also change this logic?